savant/deepstream/drawfunc.py

```diff
@@ -41,20 +40,11 @@ def __init__(self, **kwargs):
             self.default_spec_track_id = get_default_draw_spec(track_id=True)
             self.default_spec_no_track_id = get_default_draw_spec(track_id=False)
 
-        self.frame_streams = []
-
-    def __call__(self, nvds_frame_meta: pyds.NvDsFrameMeta, buffer: Gst.Buffer):
```

Comment:
If necessary, fix vlf-pipeline modules that use the old interface.

Reply:
I've created MR in vlf-pipelines.